client/extensions/woocommerce/state/sites/shipping-zone-locations/actions.js

```diff
@@ -20,7 +20,7 @@ export const fetchShippingZoneLocations = ( siteId, zoneId ) => ( dispatch, getS
 		return;
 	}
 
-	//0 is Rest of the World zone, skip fetching locations
+	//0 is Locations not covered by your other zones zone, skip fetching locations
```

Comment:
Every match of `Rest of the World` you changed is tests or comments. I personally find "Rest of the World" better for understanding what it means (mainly because it's shorter).

To actually change what's displayed in the UI, we need to either change the output of the WC-Core API (`Rest of the World` is still the name of the zone), or put a 2-line logic in the reducer that parses the API response. I'll do the second option, but please leave the comments nice and short :)